docs/includes/openapi/openapi-generator.adoc

```diff
@@ -0,0 +1,543 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+// tag::preamble[]
+:toc:
+:toc-placement: preamble
+:description: Helidon {flavor-uc} OpenAPI Generator
+:keywords: helidon, {flavor-lc}, openapi, generator
+:feature-name: Helidon {flavor-uc} OpenAPI Generator
+// DO NOT CHANGE THE FOLLOWING - it's used as a minimum release that will not normally change with new releases of the OpenAPI generator
+:first-version-with-strong-helidon-support: 6.2.1
+// Update the following when it is convenient to keep pace with the latest releases of the OpenAPITools generator
+// end::preamble[]
+
+// tag::intro[]
+== Contents
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Configuration, Configuration>>
+- <<Usage, Usage>>
+- <<References, References>>
+
+== Overview
+The link:{openapi-spec-url}[OpenAPI specification] provides a standard way to express RESTful APIs.
+
+Separately, the link:{openapi-generator-tool-site-url}[OpenAPI generator] project has created a powerful code generator tool which accepts an OpenAPI document and generates client and server code for many languages and frameworks. The Helidon team contributes to this tool to ensure that it provides strong support for Helidon {flavor-uc} clients and servers.
+As a result, you can use the generator to create code that fits smoothly into your Helidon applications.
+The OpenAPI generator gained particularly strong support for Helidon in release
+{first-version-with-strong-helidon-support}.
+This document applies to that release and later ones.
+
+In the vocabulary of the tool, there are two _generators_ for Helidon:
+
+* `java-helidon-client` (which this document refers to as the Helidon client generator)
+* `java-helidon-server` (Helidon server generator).
+
```

Comment:
Use same structure in parenthesis for parallelism? For example, "hereinafter Helidon ... generator".

Reply:
Fixed